src/numerics/type_vector.C

```diff
@@ -65,8 +65,6 @@ template <typename T>
 void TypeVector<T>::write_unformatted (std::ostream & os,
                                        const bool newline) const
 {
-  libmesh_assert (os);
```

Comment:
Wow, you'd think at *some* point we'd stop finding little Easter eggs like this...